src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -290,8 +289,8 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                             if (refreshAll)
                             {
                                 // Trigger a single load-all operation if a change was detected in one or more key-values with refreshAll: true
-                                applicationSettings = await LoadSelectedKeyValues(client, cancellationToken).ConfigureAwait(false);
-                                watchedSettings = await LoadKeyValuesRegisteredForRefresh(client, applicationSettings, cancellationToken).ConfigureAwait(false);
+                                mappedData = await LoadSelectedKeyValues(client, cancellationToken).ConfigureAwait(false);
+                                watchedSettings = await LoadKeyValuesRegisteredForRefresh(client, mappedData, cancellationToken).ConfigureAwait(false);

```

Comment:
You're passing a parameter here named `mappedData`, but that's not really mapped data right?

Reply:
It was originally named data but that meant having an extra variable defined at the top of the method, so now it's reusing mappedData. It is correct though that it's not mapped at this point, is that too misleading?

Reply:
alright, I can change that back